frontend/packages/kubevirt-plugin/src/plugin.tsx

```diff
@@ -63,30 +64,30 @@ const plugin: Plugin<ConsumedExtensions> = [
     },
   },
   {
-    // NOTE(yaacov): vmtemplates is a template resource with a selector.
```

Comment:
Swapped between the items because now both merge before `Deployments` when its non-openshift cluster, and we still want to display VM before VMT

Reply:
+1